api/src/Entity/ContentNode/ColumnLayout.php

```diff
@@ -25,17 +27,17 @@
         'post' => [
             'denormalization_context' => ['groups' => ['write', 'create']],
             'security_post_denormalize' => 'is_granted("CAMP_MEMBER", object) or is_granted("CAMP_MANAGER", object)',
-            'validation_groups' => ColumnLayoutGroupSequence::class,
+            'validation_groups' => ColumnLayoutPostGroupSequence::class,
         ],
     ],
     itemOperations: [
         'get' => ['security' => 'is_granted("CAMP_COLLABORATOR", object) or is_granted("CAMP_IS_PROTOTYPE", object)'],
         'patch' => [
             'denormalization_context' => ['groups' => ['write', 'update']],
             'security' => 'is_granted("CAMP_MEMBER", object) or is_granted("CAMP_MANAGER", object)',
-            'validation_groups' => ColumnLayoutGroupSequence::class,
+            'validation_groups' => ColumnLayoutPatchGroupSequence::class,
         ],
-        'delete' => ['security' => 'is_granted("CAMP_MEMBER", object) or is_granted("CAMP_MANAGER", object)'],
+        'delete' => ['security' => '(is_granted("CAMP_MEMBER", object) or is_granted("CAMP_MANAGER", object)) and object.owner === null'], // disallow delete when contentNode is a root node
```

Comment:
das gehört wahrscheinlich eher in die validation, aber da stossen wir wieder auf dieses Problem: https://github.com/ecamp/ecamp3/pull/2179#discussion_r741118906